examples/hacker-news/app/StoryDetails.re

```diff
@@ -0,0 +1,81 @@
+open Brisk_macos;
+open Brisk.Layout;
```

Comment:
We have to sort out the Style situation. All of this crap should be in a `Style` module. + we should remove the functorization.